### PR TITLE
add type-check frontend script

### PR DESCRIPTION
**Pull request by dario-piotrowicz**
add such prelint script so that it runs tsc making sure that all typings are valid

### CODE REVIEW

File: backend/src/database/neon.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import type { Env } from 'wildebeest/backend/src/types/env'
 
 function sqliteToPsql(query: string): string {
 	let c = 0
-	return query.replaceAll(/\?([0-9])?/g, (match: string, p1: string) => {
+	return query.replace(/\?([0-9])?/g, (match: string, p1: string) => {
```
Comment (xtuc) [inline backend/src/database/neon.ts:7]: why this change?

Comment (dario-piotrowicz) [reply backend/src/database/neon.ts:7]: it produces a typings error:
Screenshot 2023-03-06 at 15 44 47


But anyways if the regex ends with `/g` then it applies the replacement to all the instances so I think this is ok, do you think it's not?

Comment (xtuc) [reply backend/src/database/neon.ts:7]: Yeah make sense.
